js/main.js

```diff
@@ -6,7 +6,7 @@ function setUserGreeting(username, firstName) {
     const msgGreetMsgEl = document.querySelector(".user-greet-msg");
     const userImgEl = document.querySelector(".user-profile-pic");
 
-    const greetMsg = `Hello, ${firstName}!`;
+    const greetMsg = `${firstName}!`;
```

Comment:
This will now cause the output to look like `Hello,Vividh` instead of `Hello, Vividh`
(talking about the space after the comma)

Reply:
Ohh I see, I'll fix it asap

Reply:
This is how it look like right now.. is it fine ? 

![hellotxt](https://user-images.githubusercontent.com/61474917/123260859-b0aaa380-d513-11eb-9846-4df6d88cf5b2.PNG)

Reply:
Yes 🙂